docs/BIDS.md

```diff
@@ -3,11 +3,10 @@
 
 ## Introduction
 
-[**BIDS**](http://bids.neuroimaging.io/) (Brain Imaging Data Structure) is the standard adopted for the organisation
- of the datasets used by Clinica pipelines through the command line.
- BIDS provides a unified structure for organising and describing neuroimaging and behavioural data.
- The use of a standard like BIDS makes easier developing and distributing code that uses neuroimaging datasets.
- For this reason, when using Clinica pipelines from the command line, the input format of the dataset is required to be BIDS-compliant.
+[**BIDS**](http://bids.neuroimaging.io/) (Brain Imaging Data Structure) is the standard adopted for the organisation of the datasets used by Clinica pipelines through the command line.
+[BIDS](http://bids.neuroimaging.io/) provides a unified structure for organising and describing neuroimaging and behavioural data.
+The use of a standard like [BIDS](http://bids.neuroimaging.io/) makes easier developing and distributing code that uses neuroimaging datasets.
+For this reason, when using Clinica pipelines from the command line, the input format of the dataset is required to be [BIDS](http://bids.neuroimaging.io/)-compliant.
```

Comment:
I understand that we want to be sure people get to the BIDS documentation, but I feel this is a bit overkill ? I'd keep the first one in every paragraph ?

Reply:
I agree with your maintenance point. Having a lot of links in doc pages isn't shocking to me, but I suppose this is purely subjective. 

With `rst` you can label a given link and use the label in the page content. This enables you to define the actual value of the link only once. I'm not sure we can do the same thing with markdown. I'll check that and remove some of these links if not.